### ISSUE TITLE
Merging/Cloning Options (v4.0.0) breaks 

### ISSUE THREAD

**Issue by felixheck** [pinojs/hapi-pino#46]
![image](https://user-images.githubusercontent.com/14093205/34765026-ab3d20d6-f5f0-11e7-923b-1e629e03ceab.png)

After updating the hapi-pino to 4.0.0 I get this errors.
Seems the `Hoek.merge` results in some kind of loop so that the maximum call stack is exceeded.

that works like a charm for me instead:
```
Object.assign(Hoek.clone(options),{ stream: options.stream, instance: options.instance })
```

**Comment by mcollina** [comment]
Can you send a PR with a unit test, so we avoid this regression?

**Comment by felixheck** [comment]
of course :)

**Comment by mcollina** [comment]
Fixed in #47
